oms-global/docker/traefik/traefik.toml.dev

```diff
@@ -94,7 +94,17 @@ swarmmode = false
   [frontends.jira-cors-frontend.routes.main]
     rule = "PathPrefixStrip:/services/jira-cors/;"
 
+  [frontends.calendar-cors-frontend]
+  backend = "calendar-cors-backend"
+  priority = 20
+  [frontends.calendar-cors-frontend.routes.main]
+    rule = "PathPrefixStrip:/services/calendar-cors/;"
+
 [backends]
   [backends.jira-cors-backend]
     [backends.jira-cors-backend.servers.server1]
     url = "https://oms-project.atlassian.net/"
```

Comment:
@serge1peshcoff / @LeonVreling one of you could have updated this one :D

Reply:
ah crap, indeed. This shouldn't be critical as we have redirect there right?

Reply:
yes idk for how long (any organisation could claim that name, and redirect would break of course)

Reply:
@LeonVreling can you make a PR and update it?